### PR TITLE
Initial version of a grafana service definition

### PR DESCRIPTION
**Pull request by jwende**
Initial version of a grafana service definition

### CODE REVIEW

**Comment by joewxboy** [comment]
@jwende It looks like you forgot to add the `-s` signify flag to your commit.  Can you amend it to add it?
Something like `commit -s --amend`

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Multi-arch docker container instance of the open-source nodered project intended for Open Horizon Linux edge nodes
```
Comment (joewxboy) [inline Makefile:1]: Is this Multi-arch, and likely change "modeled" to "grafana"

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Repository Maintainers
+======================
+
+See the information about [community membership roles](https://wiki.lfedge.org/display/OH/Community+Membership) to learn about the role of the maintainers and the process to become one.
+
+| Name         | GitHub      | email                    |
+| ------------ | ----------- | ------------------------ |
+| Joe Pearson  | joewxboy    | <joe.pearson@us.ibm.com> |
```
Comment (joewxboy) [inline MAINTAINERS.md:8]: You can add your name to this list, if you'd like, since you contributed these files.

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+
+## Prerequisites
+
+**Management Hub:** [Install the Open Horizon Management Hub](https://open-horizon.github.io/quick-start) or have access to an existing hub in order to publish this service and register your edge node.  You may also choose to use a downstream commercial distribution based on Open Horizon, such as IBM's Edge Application Manager.  If you'd like to use the Open Horizon community hub, you may [apply for a temporary account](https://wiki.lfedge.org/display/LE/Open+Horizon+Management+Hub+Developer+Instance) and have credentials sent to you.
+
+**Edge Node:** You will need an x86 computer running Linux or macOS, or a Raspberry Pi computer (arm64) running Raspberry Pi OS or Ubuntu to install and use Node-Red deployed by Open Horizon.  You will need to install the Open Horizon agent software, anax, on the edge node and register it with a hub.
```
Comment (joewxboy) [inline README.md:10]: Change "Node-Red" to "Grafana"

File: README.md [hunk 4]
```diff
@@ -1 +1,107 @@
...
+
+## Installation
+
+Clone the `service-grafana` GitHub repo from a terminal prompt on the edge node and enter the folder where the artifacts were copied.
+
+  NOTE: This assumes that `git` has been installed on the edge node.
+
+  ``` shell
+  git clone https://github.com/jwende/service-grafana.git
+  cd service-nodered
```
Comment (joewxboy) [inline README.md:22]: "cd service-nodered" should be "cd service-grafana"

File: README.md [hunk 5]
```diff
@@ -1 +1,107 @@
...
+
+`make check` to see the values in your environment variables and how they compare to the default values.  It will also show the service definition file with those values filled in.
+
+`make deploy-check` to see if the properties and constraints that you've configured match each other to potentially form an agreement.
+
+`make test` to see if the web server is responding.
+
+`make attach` to connect to the running container and open a shell inside it.
+
+> **Note** The Node-Red container by default runs in un-privileged mode, but it may require privileged conditions in certain cases. In that case you can manually add "--privileged" flag in the Makefile under `docker-run` command.
```
Comment (joewxboy) [inline README.md:80]: "Node-Red" should be "Grafana"

**Review by joewxboy — CHANGES_REQUESTED** [review]
A couple of instances found referring to Node-Red in the files.

**Comment by jwende** [comment]
I'll make the changes in the original repo and create the pull request again.
J.

**Comment by jwende** [comment]
ok - I've tried and will create a new pull request. 

J.




Gesendet: Sonntag, 03. März 2024 um 15:15 Uhr
Von: "Joe Pearson" ***@***.***>
An: "open-horizon-services/service-grafana" ***@***.***>
Cc: "Joerg" ***@***.***>, "Mention" ***@***.***>
Betreff: Re: [open-horizon-services/service-grafana] Initial version of a grafana service definition (PR #3)


 

@jwende It looks like you forgot to add the -s signify flag to your commit. Can you amend it to add it?
Something like commit -s --amend

**Pull request closed without merging**